### PR TITLE
[TASK] Use type casts instead of converter functions

### PR DESCRIPTION
**Pull request by oliverklee**
This will improve performance.

### CODE REVIEW

File: api/class.templatemethods.php [hunk 1]
```diff
@@ -472,8 +472,7 @@ public function method_strftime($mData, $aParams)
         }
 
         return strftime(
-            $sFormat,
-            intval(trim($this->templateDataAsString($mData)))
+            $sFormat, (int)trim($this->templateDataAsString($mData))
```
Comment (hannesbochmann) [inline api/class.templatemethods.php]: Please correct the the CS. Only one argument is allowed per line in a multi-line function call.

File: util/class.tx_mkforms_util_FormBase.php [hunk 2]
```diff
@@ -559,7 +559,8 @@ public static function addFields(array $data, array $fields)
                     unset($data[$fieldKey]);
                 } // Den Wert setzen oder Überschreiben
                 elseif (!$bUnset) {
-                    $bOverride = isset($fields[$fieldKey.'.']) && isset($fields[$fieldKey.'.']['override']) && (intval($fields[$fieldKey.'.']['override']) > 0);
+                    $bOverride = isset($fields[$fieldKey.'.']) && isset($fields[$fieldKey.'.']['override']) && ((int)$fields[$fieldKey
+                            . '.']['override'] > 0);
```
Comment (hannesbochmann) [inline util/class.tx_mkforms_util_FormBase.php]: Please remove the new line break or add line breaks after each part of the condition.

File: api/class.templatemethods.php (hunk 1 above)
Comment (oliverklee) [reply api/class.templatemethods.php]: Done.

File: util/class.tx_mkforms_util_FormBase.php (hunk 2 above)
Comment (oliverklee) [reply util/class.tx_mkforms_util_FormBase.php]: Done.

**Comment by oliverklee** [comment]
Updated, rebased and repushed.

**Pull request merged**